monkey/monkey_island/cc/services/attack/technique_reports/__init__.py

```diff
@@ -94,6 +109,8 @@ def get_message_and_status(cls, status):
         """
         return {"message": cls.get_message_by_status(status), "status": status}
 
+    config_schema_per_attack_technique = None
+
```

Comment:
If this is a class variable, it should probably go at the top of the class instead of floating somewhere between class methods. Is there any reason for it to be a class variable instead of belonging to the method? As a class variable, it seems improperly scoped, as its only used in `get_message_by_status()`

Reply:
Doing what we discussed yesterday leads to this:
![image](https://user-images.githubusercontent.com/44770317/135032849-3f9060dd-5a5e-426a-9bc4-6cc21ee2c5aa.png)